integration_tests/tests/test_logging/containerpilot.json5

```diff
@@ -0,0 +1,43 @@
+{
+  consul: "consul:8500",
+  logging: {
+    level: "DEBUG",
+    format: "text"
+  },
+  jobs: [
+    {
+      name: "job1",
+      // this echo gets wrapped in a log line
+      exec: "echo job1 exec",
+    },
+    {
+      name: "job2",
+      // this echo is not wrapped in a log line
+      exec: "echo job2 exec",
+      passthru: true,
```

Comment:
I'm not crazy about the word `passthru` but unable to think of a better alternative for this feature.